src/Symfony/Bundle/FrameworkBundle/Resources/config/messenger.xml

```diff
@@ -34,6 +34,11 @@
             <argument type="service" id="validator" />
         </service>
 
+        <service id="messenger.middleware.record_messages" class="Symfony\Component\Messenger\Middleware\HandlesRecordedMessagesMiddleware" abstract="true">
+            <argument type="service" id="message_bus" />
```

Comment:
`message_bus`? Will you dispatch the events into the same bus than the command?

Reply:
This will be overwritten by your configuration. It is just abstract.